pandas/_libs/tslib.pyx

```diff
@@ -691,9 +691,7 @@ cpdef array_to_datetime(ndarray[object] values, errors='raise',
                             continue
                         elif require_iso8601:
                             if is_raise:
-                                raise ValueError("time data {val} doesn't "
-                                                 "match format specified"
-                                                 .format(val=val))
+                                continue
```

Comment:
Don't you mean to do `raise` here?

Reply:
Also, don't you have to make a similar change [here] as well?

Reply:
I take by [here] you meant line 626 in `tslib.pyx`?

Reply:
Yep, sorry.  I forgot to add the link.